microsoft-edge/web-platform/site-impacting-changes.md

```diff
@@ -5,19 +5,19 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 01/24/2022
+ms.date: 11/23/2022
 ---
 # Site compatibility-impacting changes coming to Microsoft Edge
 
 This article lists differences between the schedule of changes for Microsoft Edge versus the Chromium project, and high-impact changes which the Microsoft Edge team is tracking especially closely.
+  
+The web platform is a collection of technologies used for building webpages, including HTML, CSS, JavaScript, and many other open standards.  The web platform constantly evolves to improve the user experience, security, and privacy.  In some cases, these changes may affect the functionality of existing webpages.
 
-The web platform is a collection of technologies used for building webpages, including HTML, CSS, JavaScript, and many other open standards.  The web platform constantly evolves to improve the user experience, security, and privacy.  In some cases, changes may affect the functionality of existing webpages.
-
-For functionality and compatibility reasons, Microsoft Edge adopts nearly all of the Chromium project's changes to the web platform.  Microsoft remains in full control of the Microsoft Edge browser and may defer or reject changes.  The Microsoft Edge team decides if the change benefits browser users.
+For functionality and compatibility reasons, Microsoft Edge adopts nearly all of the Chromium project's changes to the web platform.  Microsoft retains full control of the Microsoft Edge browser and may defer or reject changes.  Then, the Microsoft Edge team decides if the change benefits browser users.
```

Comment:
```suggestion
For functionality and compatibility reasons, Microsoft Edge adopts nearly all of the Chromium project's changes to the web platform.  However, Microsoft retains full control of the Microsoft Edge browser and may defer or reject changes. 
 The Microsoft Edge team decides if the change benefits browser users.
```

I don't think adding "then" is correct here. Microsoft may decide to reject changes specifically because they may not benefit our users.